### PR TITLE
Add documentation for push notification metadata

### PR DESCRIPTION
**Pull request by bitsofdave**

### CODE REVIEW

File: guides/notifications.md [hunk 1]
```diff
@@ -31,6 +31,76 @@ The following events are currently supported:
   6.  Time off responded
 
 
+Push Notification Metadata
+--------------------------
+The metadata sent with the push notification depends on the type of notification:
+
+  1.  @mention push notifications:
+      ```json
```
Comment (macat) [inline guides/notifications.md]: @bitsofdave Formatting of these blocks is messed up. I think you shouldn't indent these lines. 

api-specs 2016-12-16 12-58-30

Comment (macat) [inline guides/notifications.md]: Well, you need to make those fields strings as json doesn't support this syntax.

File: guides/notifications.md [hunk 2]
```diff
@@ -31,6 +31,76 @@ The following events are currently supported:
...
 
 
+Push Notification Metadata
+--------------------------
+The metadata sent with the push notification depends on the type of notification:
+
+1.  @mention push notifications:
+    ```json
+    {
+      notification_id: "<integer>",
```
Comment (macat) [inline guides/notifications.md]: You need to make these fields strings as json doesn't support this syntax.

**Comment by bitsofdave** [comment]
@macat Updated. Should look better now.

**Pull request merged**